lib/oli_web/controllers/dev_controller.ex

```diff
@@ -0,0 +1,13 @@
+defmodule OliWeb.DevController do
+  use OliWeb, :controller
+
+  alias Oli.Utils.FlameGraph
+
+  def flame_graphs(conn, _params) do
```

Comment:
Displays all SVG urls, and when invoked, rebuilds the actual SVGs from the raw stack data for each file